dali_tf_plugin/daliop.cc

```diff
@@ -20,7 +20,11 @@
 
 // for Eigen::GpuDevice
 #define EIGEN_USE_GPU
+#if TF_MAJOR_VERSION == 2 && TF_MINOR_VERSION >= 16
 #include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+#else
+#include "unsupported/Eigen/CXX11/Tensor"
+#endif
```

Comment:
I don't understand this piece. It seems that the new build path affects only the older versions. Before this change the only build path was the one that now seems to be restricted to the newer versions. How did it work before and why would it stop working now?

Reply:
Worked by accident. Fixed.